software/Test_Robot_Software/api.py

```diff
@@ -0,0 +1,51 @@
+import math
+
+import sys
+import time
+import struct
+
+import serial
+s = serial.Serial("/dev/ttyACM0", baudrate=115200)
```

Comment:
Serial port should be discovered dynamically. Refer to it here https://ut-robotics.github.io/picr22-home/basketball_robot_guide/software/review_summary.html#_use_serial_tools_list_ports_or_equivalent_to_find_correct_serial_port

Reply:
Code updated to find the available serial port dynamically